.circleci/config.yml

```diff
@@ -37,6 +37,11 @@ jobs:
           cd focalboard
           git checkout $CIRCLE_BRANCH || git checkout <<parameters.target-branch>> || git checkout rolling-stable
           echo $(git rev-parse HEAD)
+          cd ..
```

Comment:
is circleci still sth? should I migrate this work to github actions?

Reply:
I think we can skip everything related to CI for part 1 of this effort. We don't intend to ship Playbooks in MPA without Monorepo, and everything changes with the latter.

Reply:
I'm working on the migration, yes we can ignore for now.